apps/api/src/domain/models/media-item/entities/media-item.entity.ts

```diff
@@ -42,12 +41,13 @@ export class MediaItem extends Resource implements ITimeBoundable {
     })
     readonly titleEnglish?: string;
 
-    @NestedDataType(ContributorAndRole, {
-        isArray: true,
-        label: 'contributions',
-        description: 'acknowledgement of those who worked on creating and producing the media item',
-    })
-    readonly contributorAndRoles: ContributorAndRole[];
+    // @NestedDataType(ContributorAndRole, {
```

Comment:
Remove comment?

Reply:
we want it there until we do a migration.